profile/README.md

```diff
@@ -1,32 +1,43 @@
-# Arcalot
-
-<img align="right" width="200px" style="padding-left: 2em;" alt="Arcalot logo showing a shield with the Arcalot inscription on a hill with the silhouette of a castle in the background" src="https://github.com/arcalot/.github/raw/main/branding/arcalot.png">
-
-The Arcalot community develops tools, plugins, and libraries that you can use either standalone as a library, and/or via a user interface or CLI. You can run the tools locally, remotely, or as part of a bigger system. Arcalot:
-
-* Helps you create workflows with **normalized input and output schemas**
-* Provides you with **assisted and automated root cause analysis** for the workflows you create as well as CI and other log systems
-* Provides **stable plugins for several workloads**
+# <a href="https://arcalot.io">Arcalot</a>
```

Comment:
I do like having the title and image link ...

But why are you using raw HTML instead of Markdown here? The Markdown would render the same, and the "source" would look more natural: `# [Arcalot](https://arcalog.io)`

(You can link the image with `[<img ...>](https://arcalot.io)`, although with all your `img` tag options I'm not sure that'd be a whole lot easier to read...)

Reply:
I decided to remove the links from the headings.

Reply:
That's too bad: I like the links. Ah, well.

Reply:
Re-added. ;)